download-wgpu-native.py

```diff
@@ -86,7 +95,14 @@ def main(version, os_string, arch, upstream, build):
     current_version = get_current_version()
     if version != current_version:
         print(f"Version changed, updating {VERSION_FILE}")
-        write_current_version(version)
+        filename = "commit-sha"
+        url = f"https://github.com/{upstream}/releases/download/v{version}/{filename}"
+        commit_sha_filename = os.path.join(tmp, filename)
+        print(f"Downloading {url} to {commit_sha_filename}")
+        download_file(url, commit_sha_filename)
+        with open(commit_sha_filename) as fh:
+            commit_sha = fh.read().strip()
```

Comment:
Or `commit_sha = requests.get(url).read().decode().strip()`

Reply:
Mmm shorter indeed. Will keep that in mind for next time :)